dashboards/datadog_audit_trail_overview/README.md

```diff
@@ -0,0 +1,28 @@
+# Audit Trail Overview
+
+Understand how your end-users use Datadog by leveraging Audit Trail.
```

Comment:
Can we add a bit more of information here, please? Maybe explaining what Audit trail is, and a link to docs?